src/extensions/ExtensionManager.as

```diff
@@ -323,6 +329,9 @@ public class ExtensionManager {
 			if(b.isRequester) {
 				if(b.requestState == 2) {
 					b.requestState = 0;
+					// TLF: should it increase the blocking count here instead...?
+					// If so then it'll need also increasing in resetArgsBlockingCount
+					// for any blocks which are requesters.
```

Comment:
OK, IIRC it looks like the original code was causing a yield also in the case of zero count - was that unnecessary, then?